qt/applications/workbench/workbench/app/mainwindow.py

```diff
@@ -395,19 +395,21 @@ def _discover_python_interfaces(self, interface_dir):
 
         return interfaces
 
-    def _discover_cpp_interfaces(self):
+    def _discover_cpp_interfaces(self, Interfaces):
```

Comment:
Can we following the coding conventions and surrounding code and lower-case the `I`? Can we also update the method docstring to include a description of the argument?